ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1243,6 +1255,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-longduration-reliability-test-f14
+  cron: 31 18 14,30 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      TEST_SCENARIOS_DISASTERRECOVERY: "67718"
```

Comment:
I wonder, if these added jobs are only for OCP-67718 case, do you have other cases which need to add more new ci jobs ?

Reply:
Currently, only for OCP-67718, because the case requires we restart the cluster 20-50 times. We don't have other similar cases.

Reply:
Got it.